Sources/EpoxyBars/BarModel/BarModel.swift

```diff
@@ -157,6 +157,10 @@ extension BarModel: StyleIDProviding {}
 
 extension BarModel: WillDisplayProviding {}
 
+// MARK: DidSelectProviding
+
+extension BarModel: DidSelectProviding {}
```

Comment:
Can we add a documentation comment that this only has an effect within a `SelectableStackView`?